keptn-cert-manager/eventfilter/eventfilter_test.go

```diff
@@ -76,3 +77,32 @@ func Test_matchesName(t *testing.T) {
 
 	assert.False(t, matchesName(deployment, []string{"my-deployment"}))
 }
+
```

Comment:
Good test for the match label! ForLabelsAndNamespace and ForNamesAndNamespace could also be easily covered :)